themes/finna2/templates/librarycards/home.phtml

```diff
@@ -76,12 +76,12 @@
                   <?php endif; ?>
                   <a class="btn-link" href="<?=$this->url('editLibraryCard') . $this->escapeHtmlAttr($record['id']) ?>" title="<?=$this->transEsc('Edit Library Card')?>"><i class="fa fa-pen gray" aria-hidden="true"></i> <?=$this->transEsc('Update Card Details')?></a>
                   <div class="dropdown">
-                    <a class="btn-link dropdown-toggle" data-toggle="dropdown" href="<?=$this->url('librarycards-deletecard') ?>?cardID=<?=urlencode($record['id'])?>">
```

Comment:
Hyvä huomio, mutta escapeHtmlAttr pitäisi olla koko attribuutin ympärillä. urlencode:n voisi samalla muuttaa niin, että käytetään url-helperin toimintoja. Eli kokonaisuus:
`<?=$this->escapeHtmlAttr($this->url('librarycards-deletecard', [], ['query' => ['cardID' => $record['id']]]))?>`